spec/components/standup_meeting_group/check_in_button.html.erb

```diff
@@ -0,0 +1 @@
+<%= link_to 'Check In', standup_meeting_group_standup_meetings_path(@standup_meeting_group), class: "btn btn-primary"%>
```

Comment:
Both of these `spec/` files don't really need to exist if you're not testing anything here.

Reply:
@JoshDevHub  "If that sounds worth pursuing to you, I'd be glad to pair on it and help out." Uhhh. Yes please.